heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
That's a good point. Please do it.

Kindly,

Runhang

> On Aug 5, 2016, at 08:20, Prabhu Inbarajan <notifications@github.com> wrote:
> 
> In heron/common/src/python/argparser.py:
> 
> > @@ -0,0 +1,196 @@
> > +# Copyright 2016 Twitter. All rights reserved.
> I am wondering if we should also name the class to be more generic , say HeronArgumentParser instead of making it specific to HeronRC. this also provides the fallback behavior in case the RC file is not found.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>